### DISCUSSION TITLE
Recognize App in aio

### DISCUSSION THREAD

**Discussion by laha83** [nextcloud/all-in-one#2276] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
I would like too integrate the recognize App, but unfortunatly I am struggling to integrate the recommended tmpfs /tmp Volume to the nextcloud Container.
I am running NC-aio ob Ubuntu 22.04 with docker and i am using docker Compose to deploy the mastercontainer

See above
### Describe the solution you'd like

Better dokumentation or enhancement of the dokumentation 

### Describe alternatives you've considered

Read through several GitHub and other sources

**Comment by szaimen** [comment]
Hi, a tmpfs volume should not be necessary

**Comment by laha83** [comment]
But why is it then described here to make use of such a tmpfs /tmp mount?🤷
[https://github.com/nextcloud/recognize](https://github.com/nextcloud/recognize)

**Comment by szaimen** [comment, marked as answer]
Since AIO [sets the tempdirectory to a folder in the datadir](https://github.com/nextcloud/all-in-one/blob/81a8352d6479920aa57cb9ac260fe69dda84a5f8/Containers/nextcloud/entrypoint.sh#L412) due to recommendation for big file uploads, it does not make sense to add such a tmpfs here.

**Comment by szaimen** [comment]
Do you possibly run AIO on a arm64 device? Apparently you'll need to add nodejs to the container manually by using this feature: https://github.com/nextcloud/all-in-one#how-to-add-os-packages-permanently-to-the-nextcloud-container

See https://github.com/nextcloud/recognize/issues/784
